service/src/main/java/bio/terra/pipelines/service/ImputationService.java

```diff
@@ -70,43 +74,38 @@ public class ImputationService {
    * successful.
    *
    * @param userId
-   * @param pipelineVersion
+   * @param pipelineId
    * @return String jobId
    *     <p>Note that the information in the requested job will grow over time, along with the
    *     following related classes:
    * @see ImputationJob
    */
   public UUID createImputationJob(
-      UUID jobId,
-      String userId,
-      String description,
-      String pipelineVersion,
-      Object pipelineInputs) {
-    logger.info("Create new imputation version {} job for user {}", pipelineVersion, userId);
+      UUID jobId, String userId, String description, Long pipelineId, Object pipelineInputs) {
+    logger.info("Create new {} job for user {}", PipelinesEnum.IMPUTATION_MINIMAC4, userId);
 
     JobBuilder jobBuilder =
         jobService
             .newJob()
             .jobId(jobId)
             .flightClass(RunImputationJobFlight.class)
-            .addParameter(JobMapKeys.PIPELINE_ID.getKeyName(), PipelinesEnum.IMPUTATION)
+            .addParameter(JobMapKeys.PIPELINE_NAME.getKeyName(), PipelinesEnum.IMPUTATION_MINIMAC4)
```

Comment:
same question here for whether we want to hardcode minimac in the generic ImputationService